tests/MailChimp/NewsletterTest.php

```diff
@@ -264,4 +264,25 @@ public function is_can_create_a_campaign()
 
         $this->newsletter->createCampaign($fromName, $replyTo, $subject, $html, $listName, $options, $contentOptions);
     }
+
+    /** @test */
+    public function it_will_throw_an_exception_when_using_a_subscribe_status_that_is_invalid()
```

Comment:
That's a nice test. Could you add another one to make sure that when `status` gets passed in with the `$options` that I'll override the default?